detectron2/structures/boxes.py

```diff
@@ -311,6 +311,29 @@ def __iter__(self):
         yield from self.tensor
 
 
+def pairwise_intersection(
+    boxes1: Boxes, boxes2: Boxes
+) -> Tuple[torch.Tensor, torch.Tensor, torch.Tensor]:
+    """
+    Given two lists of boxes of size N and M,
+    compute the intersection area between __all__ N x M pairs of boxes.
+    The box order must be (xmin, ymin, xmax, ymax)
+    Args:
```

Comment:
We follow the [google style](https://www.sphinx-doc.org/en/master/usage/extensions/napoleon.html) and a new line is required before `Args:` here so it can be parsed properly